src/Orchard.Web/Modules/Orchard.DynamicForms/Tokens/FormTokens.cs

```diff
@@ -9,14 +9,17 @@ public class FormTokens : Component, ITokenProvider {
             context.For("FormSubmission", T("Dynamic Form submission"), T("Dynamic Form Submission tokens for use in workflows handling the Dynamic Form Submitted event."))
                 .Token("Field:*", T("Field:<field name>"), T("The posted field value to access."), "Text")
                 .Token("IsValid:*", T("IsValid:<field name>"), T("The posted field validation status."))
+                .Token("FormName", T("FormName"), T("The posted form's name."))
```

Comment:
"The name of the form posted." seems to be better grammatically, but I am not a native speaker so I will see others' comments.